assets/app/scripts/services/data.js

```diff
@@ -1194,12 +1202,19 @@ DataService.prototype.createStream = function(resource, name, context, opts, isR
     return new URI({protocol: protocol, hostname: hostPort}).toString();
   };
 
-  var CACHED_RESOURCE = {
+  // - *should* be hit frequently enough that the LRU does not kick out as we wont fetch again
```

Comment:
if it did kicked out of the cache then we'd watch it again anyway since hasImmutable checks if we have the data in the cache, maybe just update the comment so its clear, as it reads now it sounds like we are just hoping it never gets cleared out :)